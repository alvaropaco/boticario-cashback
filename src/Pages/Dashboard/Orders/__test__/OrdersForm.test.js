import React from 'react';
import OrdersForm from '../OrdersForm';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom'
import { createBrowserHistory } from "history";

const history = createBrowserHistory();

it('renderiza a pÃ¡gina de Compras', () => {
    const tree = renderer
    .create(
        <Router history={history}>
            <OrdersForm />
        </Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});