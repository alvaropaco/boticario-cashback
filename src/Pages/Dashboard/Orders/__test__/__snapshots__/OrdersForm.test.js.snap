// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza a página de Compras 1`] = `
<form>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
  >
    Cadastrar nova Compra
  </h6>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
    >
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Mui-required Mui-required"
          data-shrink={false}
          htmlFor="code"
          id="code-label"
        >
          Código da Compra
          <span
            aria-hidden={true}
            className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
          >
             
            *
          </span>
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="code"
            name="code"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="text"
          />
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-adornedStart"
        label="Valor total"
        onClick={[Function]}
        required={true}
      >
        <div
          className="MuiInputAdornment-root MuiInputAdornment-positionStart"
        >
          <p
            className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            $
          </p>
        </div>
        <input
          className="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedStart"
          id="amount"
          name="amount"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
        />
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
    >
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink={true}
          htmlFor="datetime"
          id="datetime-label"
        >
          Data
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiInput-input"
            defaultValue="31/08/2020"
            disabled={false}
            id="datetime"
            name="datetime"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="date"
          />
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Cadastrar
        </span>
      </button>
    </div>
  </div>
</form>
`;
